src/controller/proxy/controller.go

```diff
@@ -122,6 +123,7 @@ func (c *controller) ProxyManifest(ctx context.Context, p *models.Project, art l
 	}
 	// Push manifest in background
 	go func() {
+		ctx := orm.Context()
```

Comment:
Use `func NewContext(ctx context.Context, o orm.Ormer) context.Context` instead? Or the other information in the context will be lost

Reply:
I don't think we need to use the `NewContext(ctx context.Context, o orm.Ormer)` here. What will happen when the HTTP goroutine was finished but this goroutine still not be scheduled by the runtime of Golang?

Reply:
In current case, we need to get the user information from context, update to use the  NewContext(ctx context.Context, o orm.Ormer)

Reply:
I created an issue for the problem to get the operator of the event, https://github.com/goharbor/harbor/issues/12774.

If we don't get the operator in the `Resolve` of event metadata implicitly, I think we don't need the ctx from the HTTP request in this goroutine.

Reply:
OK, I will add Operator field in PullArtifactEventMetadata